src/agent/library/skills.js

```diff
@@ -688,6 +713,11 @@ export async function goToPosition(bot, x, y, z, min_distance=2) {
         log(bot, `Missing coordinates, given x:${x} y:${y} z:${z}`);
         return false;
     }
+    if (bot.modes.isOn('cheat')) {
+        bot.chat('/tp @s ' + x + ' ' + y + ' ' + z);
```

Comment:
Will the use of teleport ever put the bot inside of a block?

Reply:
It can. also it can place blocks on top of itself and end up in the same situation. 

but i think its fine, it can always teleport out. I don't think disallowing it would be useful